flashlib.frc.nt/src/main/java/com/flash3388/frc/nt/builtin/NtPidTable.java

```diff
@@ -0,0 +1,85 @@
+package com.flash3388.frc.nt.builtin;
+
+import com.beans.observables.ObservableDoubleValue;
+import com.flash3388.flashlib.robot.control.PidController;
+import com.flash3388.frc.nt.NtTable;
+import com.flash3388.frc.nt.beans.NtDoubleProperty;
+
+public class NtPidTable {
+    private static final String PID_TABLE_NAME = "Pid";
+    private static final String KP_ENTRY_NAME = "Kp";
+    private static final String KI_ENTRY_NAME = "Ki";
+    private static final String KD_ENTRY_NAME = "Kd";
+    private static final String KF_ENTRY_NAME = "Kf";
+    private static final String PID_INPUT_ENTRY_NAME = "Input";
+    private final NtTable mTable;
+
+    public NtPidTable(double kP, double kI, double kD, double kF, ObservableDoubleValue pidInput) {
```

Comment:
Yes of course, how hard is it to do on the robot side? Or you prefer to just give the method to update the process var and let the user call it wherever they want?

Reply:
It depends, because it's a physical sensor most likely, so it would requiring polling. It's actually difficult, you can use the `Scheduler` for that, define an action that runs always.

Reply:
Oh man I forgot about Actions! they are cool btw